lang/py/avro/test/test_schema.py

```diff
@@ -629,6 +641,10 @@ def test_invalid_name(self):
             None,
             None,
         )
+        # A name cannot start with dot."
+        self.assertRaises(avro.errors.InvalidName, avro.schema.Name, ".a", None, None)
+        self.assertRaises(avro.errors.InvalidName, avro.schema.Name, "o..a", None, None)
+        self.assertRaises(avro.errors.InvalidName, avro.schema.Name, ".a", None, None)
```

Comment:
```suggestion
        self.assertRaises(avro.errors.InvalidName, avro.schema.Name, "a.", None, None)
```
Looks like this is a repeated test, did you mean to test trailing dots?